examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -79,42 +97,15 @@ export class TaskList extends DataObject implements ITaskList {
         // To add a task, we update the root SharedDirectory.  This way the change is propagated to all collaborators
         // and persisted.  In turn, this will trigger the "valueChanged" event and handleTaskAdded which will update
         // the this.tasks collection.
+
+        this.savedData.set(id, { id, name: nameString.handle, priority: priorityCell.handle });
```

Comment:
Let's create a `PersistedTask` here to clarify typing as we store it into `savedData`, same as what is done below for `data` into `draftData`